base_product_merge/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright 2023 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```

Comment:
```suggestion
# License LGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```

Reply:
The licence changed to AGPL because of openupgrade merge tool